docs/05-plugin-development.md

```diff
@@ -257,23 +258,23 @@ augmentChunkHash(chunkInfo) {
 Type: `string | (() => string)`<br>
 Kind: `async, parallel`<br>
 Previous Hook: [`renderStart`](guide/en/#renderstart)<br>
-Next Hook: 
+Next Hook: [`augmentChunkHash`](guide/en/#augmentchunkhash) for each chunk with that would contain a hash in the file name. Otherwise [`resolveFileUrl`](guide/en/#resolvefileurl) for each use of `import.meta.ROLLUP_FILE_URL_referenceId` and [`resolveImportMeta`](guide/en/#resolveimportmeta) for all other accesses to `import.meta`. Otherwise [`renderChunk`](guide/en/#renderchunk) for each chunk.
```

Comment:
The use of "otherwise" tripped me up here. It makes it sound like `resolveFileUrl` won't happen if `augmentChunkHash` happens, but you mean "otherwise `resolveFileUrl` will happen next". Maybe I just wasn't reading closely and I'll be the only one who gets confused here.

"Otherwise" could probably be replaced with "then", or just remove those statements and only mention `resolveFileUrl` as the "next hook" to `augmentChunkHash`.

Reply:
I will use `then`